packages/react-frontend/components/Review.jsx

```diff
@@ -0,0 +1,54 @@
+import Rating from "@mui/material/Rating";
+import StarBorderIcon from "@mui/icons-material/StarBorder";
+import DeleteIcon from "@mui/icons-material/Delete";
+import EditIcon from "@mui/icons-material/Edit";
+import IconButton from "@mui/material/IconButton";
+import "./Review.css";
+import { Link } from "react-router-dom";
+
+export default function Review(props) {
+  let review = props.review;
+  let rightUser = props.rightUser;
+
+  return (
+    <div className="userReview">
+      <div className="albumInfo">
+        <img src={review.album_id.album_cover}></img>
```

Comment:
you could also add an alt attribute for accessibility

Reply:
agree!